src/Domain/DOMPurify.ts

```diff
@@ -0,0 +1,3 @@
+export const purifyConfig = {
```

Comment:
Это происходит автоматически. Я делаю `npx @khanacademy/flow-to-ts --write --delete-source src` 
```
--write write output to disk instead of STDOUT
--delete-source delete the source file
``` 
следом сразу коммит чтобы было проще прослеживать изменения. То есть тут конкретно создаются новые файлы, а не обновляются старые. Почему-то гит иногда понимает что это js -> ts, но я не понял почему

Reply:
Это правильно, что git ренеймит. Нужно всё так делать. Чтобы git история сохранялась